### DISCUSSION TITLE
Mandatory Attribute not marked as mantadory

### DISCUSSION THREAD

**Discussion by marcus-at-localhost** [atrocore/atrocore#551] [Q&A] [answered]
I've got Attributes in Attributes Groups, arranged in a Product Family.

For some reason, Attributes are displayed in _italics with an asterisk *_

![image](https://user-images.githubusercontent.com/1337223/205168586-60aa6e13-073c-4e3f-a201-8eb78bc80d5b.png)

It's not clear to my why it is marked up like that. The * means it's a mandatory field, but why is it in italic.
And why is it mandatory when I didn't set it as that?

![image](https://user-images.githubusercontent.com/1337223/205168193-0df848ad-69b1-4214-bb1b-a9f8c448a732.png)

Now anytime an attribute has no value I can't save, because it's a mantadory.

Any advice? Thanks!

**Comment by atrolex** [comment, marked as answer]
Hi Marcus,  you are writing about mandatory attribute value.  There is a field in the entity Product Attribute Value for marking that.
Whether an attribute value is mandatory or not is inherited only from the parent product, or from the product family.
Marking attribute as mandatory is treated as "default value" if you add this attribute to a product or to a product family.
If you want to mark some attribute values as mandatory, you need to do it on the level of parents or product families, i think mass update should work for that.

**Reply by marcus-at-localhost** [reply]
Thanks. I wasn't ware of that. Screenshot as reference.

![image](https://user-images.githubusercontent.com/1337223/205391614-9c682db1-b6ff-4d7d-9f07-20398724e798.png)
